cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/template/etl/realtime/ETLWorkerTest.java

```diff
@@ -249,6 +250,19 @@ public void testKafkaSource() throws Exception {
     Assert.assertEquals(3, (int) row.getInt("AGE"));
   }
 
+  private void delay(final String tableName) throws Exception {
```

Comment:
this more ``waitForTableToBePopulated``, right? Also, if you pass in the ``Table`` object, then you wouldn't have to recreate it on line 243-244